.spin/bootstrap

```diff
@@ -0,0 +1,7 @@
+#!/usr/local/env bash
+
+# Install some dependencies
+apt-get install libsqlite3-dev
```

Comment:
Not sure what is the common way to do it with `spin` but since the install is from a script it might make sense to pass the `-y` flag:

```suggestion
apt-get install -y libsqlite3-dev
```

Reply:
Good catch. Added `--no-install-recommends` as well to avoid `apt` pulling unnecessary packages. This is a similar pattern used in `Dockerfile` for Codespaces (see [here](https://github.com/microsoft/vscode-dev-containers/blob/19adbac72304efc3f7b09970a3bac3c5a68a4c27/containers/ruby/.devcontainer/Dockerfile#L10-L11)).